<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TerminaNotes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources/iconlib.qrc">
    <normaloff>:/icn/article</normaloff>:/icn/article</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>3</number>
    </property>
    <item row="0" column="1" rowspan="2">
     <widget class="QTabWidget" name="tabBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="editor">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="titleLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLineEdit" name="titleEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Comic Sans MS</family>
              <pointsize>16</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tagButton">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Tags</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollNoteArea">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="speNoteArea">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>548</width>
             <height>357</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="speNoteLayout">
            <property name="margin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonsLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="rightButtonSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="editSaveButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="centralButtonSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="deleteCancelButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Put to Trash</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="leftButtonSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="htmlBox">
       <attribute name="title">
        <string>Html</string>
       </attribute>
       <layout class="QVBoxLayout" name="htmlLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWebView" name="htmlArea">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="teXBox">
       <attribute name="title">
        <string>Tex</string>
       </attribute>
       <layout class="QVBoxLayout" name="texLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="texArea"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="txtBox">
       <attribute name="title">
        <string>Txt</string>
       </attribute>
       <layout class="QVBoxLayout" name="txtLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="txtArea"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="searchLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search By Tag :</string>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchLine">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>194</width>
            <height>323</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="noteList"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="trashLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="trashButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normalon>../icon/trash_empty.png</normalon>
            </iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>64</width>
             <height>64</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="copyright">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>30</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Projet de L021 - Semestre P13 by Jonathan Dekhtiar &amp; Florian Baune</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNewNote">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNew_Article"/>
     <addaction name="actionNew_Document"/>
     <addaction name="actionNew_Image"/>
     <addaction name="actionNew_Audio"/>
     <addaction name="actionNew_Video"/>
    </widget>
    <addaction name="menuNewNote"/>
    <addaction name="actionOpenTrash"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_WorkSpace"/>
    <addaction name="actionChange_WorkSpace"/>
    <addaction name="separator"/>
    <addaction name="actionExport_As_File"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionHtml"/>
     <addaction name="actionTeX"/>
     <addaction name="actionTxt"/>
    </widget>
    <addaction name="actionTags"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate_WorkSpace"/>
   <addaction name="actionChange_WorkSpace"/>
   <addaction name="separator"/>
   <addaction name="actionNew_Article"/>
   <addaction name="actionNew_Document"/>
   <addaction name="actionNew_Image"/>
   <addaction name="actionNew_Audio"/>
   <addaction name="actionNew_Video"/>
   <addaction name="separator"/>
   <addaction name="actionCopy_Binaries"/>
   <addaction name="separator"/>
   <addaction name="actionExport_As_File"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/quit</normaloff>:/icn/quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Close the Software</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionChange_WorkSpace">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/open</normaloff>:/icn/open</iconset>
   </property>
   <property name="text">
    <string>Change WorkSpace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate_WorkSpace">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/new</normaloff>:/icn/new</iconset>
   </property>
   <property name="text">
    <string>New WorkSpace</string>
   </property>
  </action>
  <action name="actionOpenTrash">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/full</normaloff>:/icn/full</iconset>
   </property>
   <property name="text">
    <string>Open Trash</string>
   </property>
   <property name="toolTip">
    <string>Open Trash</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit/Save</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Put to Trash/Cancel</string>
   </property>
  </action>
  <action name="actionTags">
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
  <action name="actionExport_As_File">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/export</normaloff>:/icn/export</iconset>
   </property>
   <property name="text">
    <string>Export As File</string>
   </property>
  </action>
  <action name="actionHtml">
   <property name="text">
    <string>Html</string>
   </property>
  </action>
  <action name="actionTeX">
   <property name="text">
    <string>TeX</string>
   </property>
  </action>
  <action name="actionTxt">
   <property name="text">
    <string>Txt</string>
   </property>
  </action>
  <action name="actionNew_Article">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/article</normaloff>:/icn/article</iconset>
   </property>
   <property name="text">
    <string>New Article</string>
   </property>
  </action>
  <action name="actionNew_Document">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/document</normaloff>:/icn/document</iconset>
   </property>
   <property name="text">
    <string>New Document</string>
   </property>
  </action>
  <action name="actionNew_Image">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/image</normaloff>:/icn/image</iconset>
   </property>
   <property name="text">
    <string>New Image</string>
   </property>
  </action>
  <action name="actionNew_Audio">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/audio</normaloff>:/icn/audio</iconset>
   </property>
   <property name="text">
    <string>New Audio</string>
   </property>
  </action>
  <action name="actionNew_Video">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/video</normaloff>:/icn/video</iconset>
   </property>
   <property name="text">
    <string>New Video</string>
   </property>
  </action>
  <action name="actionCopy_Binaries">
   <property name="icon">
    <iconset resource="../ressources/iconlib.qrc">
     <normaloff>:/icn/copy</normaloff>:/icn/copy</iconset>
   </property>
   <property name="text">
    <string>Copy Binaries Content</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources/iconlib.qrc"/>
 </resources>
 <connections/>
</ui>
